guides/release/components/customizing-a-components-element.md

```diff
@@ -33,6 +34,36 @@ export default Component.extend({
 </ul>
 ```
 
+Note: It is not recommended to use null tagName .
+
+The following example explains usecase when `tagName` is null:
```

Comment:
```suggestion
However, if you see `tagName: ''` in an app, it is good to know what that means. The following example explains the use case and effects.
```

Reply:
Do we mention somewhere that they are called tagless components? `tagName: null` is not valid, so we should definitely tweak that wording in the text.